### ISSUE TITLE
My lookup table keeps disappearing whenever Rasa-X edits my nlu.yml file

### ISSUE THREAD

**Issue by snek93** [RasaHQ/rasa#7998]
**Rasa version**:
2.2.10
**Rasa SDK version** (if used & relevant):
2.2.0 (used but not necessarily relevant)
**Rasa X version** (if used & relevant):
0.36.0
**Python version**:
3.7.7
**Operating system** (windows, osx, ...):
macOS
**Issue**:

My lookup tables disappear completely if they are part of my `nlu.yml` file whenever Rasa-X edits the file (since it has write access) to add any new training examples, etc. 

The only way to avoid loss was to create a new YAML in my training data folder which gets detected by Rasa-X as an available lookup table. However, trying to add some YAML files using the upload functionality in the UI does not work. 

This is only a minor bug but it caused me data-loss, which would have been problematic had I not been using version control. 

**Error (including full traceback)**:

Rasa-X displays no errors related to this when it happens. Instead, I get this unrelated error saying that my intents are in the training data but not in the domain although that's not true. 
```
Found intent 'affirm' in stories which is not part of the domain.
  More info at https://rasa.com/docs/rasa/stories
```
If relevant at all, I am also having another issue where the flow of stories displays `None` instead of the actual intent names.

**Comment by sara-tagger** [comment]
Thanks for the issue, @TyDunn will get back to you about it soon!

###### You may find help in the [docs](https://rasa.com/docs/) and the [forum,](https://forum.rasa.com/) too 🤗

**Comment by TyDunn** [comment]
@snekz Thanks for letting us know. This is a bug. I have added it to engineering backlogs

**Comment by chdorner** [comment]
This should be fixed when Rasa X 0.39 comes out.
